src/components/dev-hub/podcast-player/audio-player.js

```diff
@@ -0,0 +1,220 @@
+    width: 100%;
+`;
+
+const AudioContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    width: 100%;
+
+    @media only screen and (max-width: 600px) {
+        flex-direction: column;
+
+        ${ControlsContainer} {
+            margin-bottom: ${size.small};
+        }
+    }
+`;
+
+const StyledSliderContainer = styled('div')`
+    margin-left: ${size.default};
+    width: 100%;
+
+    @media only screen and (max-width: 600px) {
+        margin-left: 0;
+    }
+
+    @media only screen and (max-width: 480px) {
+        margin-top: 15px;
+
+        ${SliderContainer} {
+            ${mobileSliderStyles}
+        }
+    }
+`;
+
+const PlayerFooter = styled('div')`
+    margin-top: ${size.small};
+
+    @media only screen and (max-width: 600px) {
+        margin-top: ${size.medium};
+        width: 100%;
+        ${LogosContainer} {
+            > a:last-of-type {
+                margin-right: 0;
+            }
+        }
+    }
+
+    @media only screen and (max-width: 480px) {
+        ${LogosContainer} {
+            flex-direction: column;
+            > a {
+                margin-right: 0;
+                &:not(:last-of-type) {
+                    margin-bottom: 20px;
+                }
+            }
+        }
+    }
+`;
+
+const AudioPlayer = ({ podcast }) => {
+    const playerRef = useRef(null);
+    const [playerState, playerDispatch] = useReducer(
+        playerReducer,
+        playerInitialState
+    );
+    const { isPlaying, progress, duration, isReady } = playerState;
+
+    const setDuration = useCallback(duration => {
+        playerDispatch({ type: PLAYER_ACTIONS.SET_DURATION, value: duration });
+    }, []);
+
+    const setProgress = useCallback(progress => {
+        playerDispatch({ type: PLAYER_ACTIONS.SET_PROGRESS, value: progress });
+    }, []);
+
+    const toggleIsPlaying = useCallback(e => {
+        e.stopPropagation();
+        playerDispatch({
+            type: PLAYER_ACTIONS.TOGGLE_IS_PLAYING,
+        });
+    }, []);
+
+    const onChangeHandler = useCallback(
+        seconds => {
+            if (isReady) {
+                setProgress({ playedSeconds: seconds });
+                playerRef.current.seekTo(seconds);
+            }
+        },
+        [isReady, setProgress]
+    );
+
+    const onReadyHandler = useCallback(player => {
+        playerRef.current = player;
+        playerDispatch({
+            type: PLAYER_ACTIONS.SET_IS_READY,
+            value: true,
+        });
+    }, []);
+
+    const moveForwardHandler = useCallback(() => {
+        if (isReady) {
+            const { getCurrentTime, seekTo } = playerRef.current;
+            seekTo(getCurrentTime() + 30, 'seconds');
+        }
+    }, [isReady]);
+
+    const moveBackHandler = useCallback(() => {
+        if (isReady) {
+            const { getCurrentTime, seekTo } = playerRef.current;
+            seekTo(getCurrentTime() - 30, 'seconds');
+        }
+    }, [isReady]);
+
+    return podcast?.url ? (
```

Comment:
```suggestion
    return podcast ? (
```
We should be able to simplify this, was there a podcast with no URL?